netty/src/main/java/io/grpc/netty/GrpcHttp2OutboundHeaders.java

```diff
@@ -67,6 +67,16 @@ private GrpcHttp2OutboundHeaders(AsciiString[] preHeaders, byte[][] serializedMe
   }
 
   @Override
+  @SuppressWarnings("ReferenceEquality") // STATUS.value() never changes.
+  public CharSequence status() {
+    // preHeaders is never null.  It has status as the first element or not at all.
+    if (preHeaders.length >= 1 && preHeaders[0] == Http2Headers.PseudoHeaderName.STATUS.value()) {
+      return Http2Headers.PseudoHeaderName.STATUS.value();
```

Comment:
Interestingly yes, but it doesn't make the test fail.  The reason this method is called is to check for 1XX status codes, which we don't use.